src/app/components/avatar/avatar.ts

```diff
@@ -9,16 +9,16 @@ import { ChangeDetectionStrategy, Component, EventEmitter, Input, NgModule, Outp
     template: `
         <div [ngClass]="containerClass()" [class]="styleClass" [ngStyle]="style" [attr.aria-labelledby]="ariaLabelledBy" [attr.aria-label]="ariaLabel" [attr.data-pc-name]="'avatar'">
             <ng-content></ng-content>
-            <span class="p-avatar-text" *ngIf="label; else iconTemplate">{{ label }}</span>
-            <ng-template #iconTemplate><span [class]="icon" [ngClass]="'p-avatar-icon'" *ngIf="icon; else imageTemplate"></span></ng-template>
-            <ng-template #imageTemplate><img [src]="image" *ngIf="image" (error)="imageError($event)" [attr.aria-label]="ariaLabel" /></ng-template>
+            <span class="p-avatar-text" [style.position]="image ? 'absolute' : ''" *ngIf="label; else iconTemplate">{{ label }}</span>
```

Comment:
If i'm not wrong this will break the Content Security Policy. 
All the components should be use `ngStyle`. Check the PR of the @mehmetcetin01140  and @cetincakiroglu

https://github.com/primefaces/primeng/pull/16005

Reply:
That commit looks like its about XSS attacks (strict CSP) because it then might be possible to inject variables into style that shouldn't be there, since all of these changes had variables. The style here is only defining hard style values so it should be fine.

Reply:
I cannot replicate in my project where we have CSP  applied in this moment and the previous PR that I sent was reported by myself but I 90% sure your code will break the CSP.

I asked to chatGPT too and said the same and the suggestion was:

```
[ngStyle]="{'position': image ? 'absolute' : ''}"
```

but again sorry because I cannot check it properly

Reply:
I see, its more about adjusting style inline the html. I've adjusted it to match the other components.

Reply:
I think looks good now.
I hope this doesn't break anything and that the problem with CSP was terrible, very difficult to find and know what the correct solution was.